test/cmdlineTests/object_compiler/output

```diff
@@ -46,7 +46,6 @@ Text representation:
   0x00
     /* "object_compiler/input.yul":265:295   */
   return
-  pop
 stop
```

Comment:
Why is the `stop` not removed after `return`?

Reply:
This might be the safety marker between code and data.

Reply:
The safety marker is `invalid` and not `stop`, iirc, and it does not show up in the text representation.